doc/en/components/grids/_shared/search.md

```diff
@@ -9,7 +9,7 @@ namespace: Infragistics.Controls
 
 # {Platform} {ComponentTitle} Search Filter
 
-The {Platform} `{ComponentName}` search enables the process of finding values in the collection of data. We make it easier to setup this functionality and it can be implemented with search input box, buttons, keyboard navigation and other useful features for an even better user experience. While browsers natively provide content search functionality, most of the time the `{ComponentName}` virtualizes its columns and rows that are out of view. In these cases, the native grid search is unable to search data in the virtualized cells, since they are not part of the DOM. We have extended the {ProductName} Material table-based grid with a **search API** that allows you to search through the **virtualized content** of the `{ComponentName}`.
+The {Platform} {ComponentTitle} Search feature in {ProductName} enables the process of finding values in the collection of data. We make it easier to set up this functionality and it can be implemented with a search input box, buttons, keyboard navigation and other useful features for an even better user experience. While browsers natively provide content search functionality, most of the time the {ComponentTitle} virtualizes its columns and rows that are out of view. In these cases, the native grid search is unable to search data in the virtualized cells, since they are not part of the DOM. We have extended the {Platform} Material table-based grid with a **search API** that allows you to search through the **virtualized content** of the `{ComponentName}`.
```

Comment:
```suggestion
The {Platform} {ComponentTitle} Search feature in {ProductName} enables the process of finding values in the collection of data. We make it easier to set up this functionality and it can be implemented with a search input box, buttons, keyboard navigation and other useful features for an even better user experience. While browsers natively provide content search functionality, most of the time the {ComponentTitle} virtualizes its columns and rows that are out of view. In these cases, the native browser search is unable to search data in the virtualized cells, since they are not part of the DOM. We have extended the {Platform} Material table-based grid with a **search API** that allows you to search through the **virtualized content** of the `{ComponentName}`.
```

Reply:
Also correct me if I am wrong but we stopped marketing the grid as Material based some time ago and can probably omit that part of the sentence too.

Reply:
We are not marketing it here, just adding it as a buzz word and making distant relation to it.